frontend/src/metabase/setup/tests/common.unit.spec.tsx

```diff
@@ -71,20 +71,6 @@ describe("setup (OSS)", () => {
         expectSectionToHaveLabel("Add your data", "4");
         expectSectionToHaveLabel("Usage data preferences", "5");
       });
-
-      it("should not set the flag for the embedding homepage", async () => {
```

Comment:
These tests were testing the localstorage flag, that we don't have anymore.

Reply:
I added some tests that check the flags being set at the end of the file, under "embedding homepage flags"